### PR TITLE
feat(testing): port graphql integration tests to hive

### PR DESCRIPTION
**Pull request by shampoobera**

### CODE REVIEW

File: e2e/hive/simulators/generate_tests.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# this script generates the sim_tests variable containing
```
Comment (itsdevbear) [inline e2e/hive/simulators/generate_tests.sh:2]: kill the script its horrible

File: e2e/hive/simulators/graphql/testcases/12_eth_getBalance_toobig_bn.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "request": "{block (number: 33) {account(address: \"0x6295ee1b4f6dd65047762f924ecd367c17eabf8f\") { balance } }}",
+  "responses": [
+    {
+      "data": {
+        "block": {
+          "account": {
+            "balance": "0x0"
```
Comment (itsdevbear) [inline e2e/hive/simulators/graphql/testcases/12_eth_getBalance_toobig_bn.json:8]: are these from your tests or fromt he graphql tests for mainnet (like in the hive repo).

Comment (shampoobera) [reply e2e/hive/simulators/graphql/testcases/12_eth_getBalance_toobig_bn.json:8]: hive repo

File: e2e/hive/simulators/graphql/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# GraphQL Hive Integration Tests
+
+Tests were taken from https://github.com/ethereum/hive/tree/master/simulators/ethereum/graphql/testcases
+
+NOTE: these tests do not test against `hash`, `miner`, or any other state root dependent fields because we do not have a state root or `chain.rlp`. It is important to note that these do work, but there is not a good way to test them (yet) as these fields change with every test run.
```
Comment (calbera) [inline e2e/hive/simulators/graphql/README.md:5]: "... because Polaris does not have a state root. It is important ..."

**Comment by itsdevbear** [comment]
@shampoobera add the simulator to the github action matrix then g2g

**Comment by itsdevbear** [comment]
@shampoobera adding to ci caused them to start failing?

**Comment by archbear** [comment]
> @shampoobera adding to ci caused them to start failing?

the local machine wasn't running the latest polard image when using hive:test

**Pull request merged**